pkg/kube/secrets.go

```diff
@@ -92,3 +92,29 @@ func DeleteSecret(name string, namespace string, cli kubernetes.Interface) error
 		return fmt.Errorf("Failed to delete secret: %w", err)
 	}
 }
+
+func CopySecret(src string, dest string, namespace string, kubeclient kubernetes.Interface) error {
```

Comment:
Observation that secret type is not preserved in the copy (e.g. kubernetes.io/tls => Opaque). 

Should it be preserved ?

Reply:
Yes, will fix.